gsl/span

```diff
@@ -347,6 +347,7 @@ class span
 public:
     // constants and types
     using element_type = ElementType;
+    using value_type = ElementType;    
```

Comment:
Please ignore everything I said ;) I was blind, changed wrong place and didn't notice that. `remove_cv_t` works like charm :)

<img src="https://cdn.meme.am/cache/instances/folder468/57690468.jpg" width="150px">